package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "horticulturalist",
-  "version": "0.9.1",
+  "version": "0.10.0",
```

Comment:
This is the correct version for the current state of the software, but we should probably bump to 1.0.0 when it's ready for production (ie: before we release webapp 3.0.0).

Reply:
I had a plan about this, which was, roughly: 1.0.0 means it's a standalone tool that in _theory_ another company / piece of software could use.

Let me know if you feel strongly that this is a bad idea and I'll change it to whatever you like the most (it made sense to me, but semver is a giant emperor has no clothes situation anyway, so I'm not really that worried)

Reply:
I'm not really that worried either, particularly as nobody will see this number. It's just that < 1.0 always feels like alpha grade software to me because < 1.0 has special rules under semver for example, you can make breaking changes between minors. So I tend to avoid publishing anything as < 1.0.